services/comm.cpp

```diff
@@ -1859,13 +1859,14 @@ void JobDoneMsg::send_to_channel(MsgChannel *c) const
     *c << flags;
 }
 
-LoginMsg::LoginMsg(unsigned int myport, const std::string &_nodename, const std::string _host_platform)
+LoginMsg::LoginMsg(unsigned int myport, const std::string &_nodename, const std::string &_netname, const std::string _host_platform)
```

Comment:
This change would also force a min_protocol_version update.  That is not desirable.